src/examples.md

```diff
@@ -1127,6 +1127,31 @@ moving to the miniapps.**_
 <div style="clear:both;"/></div>
 <br></div>
 
+<div id="polar-nc" markdown="1">
+##Polar-NC
+<img class="floatright" src="../img/examples/polar-nc.png">
+
+This miniapp generates a circular sector mesh that consist of quadrilaterals
+and triangles of similar sizes. The 3D version of the mesh is made of prisms
+and tetrahedra.
+
+The mesh is non-conforming by design, and can optionally be made curvilinear.
+The elements are ordered along a space-filling curve by default, which makes
+the mesh ready for parallel non-conforming AMR in MFEM.
+
+The implementation also demonstrates how to initialize a non-conforming mesh
+on the fly by marking hanging nodes with `Mesh::AddVertexParents`.
+
+For more details, please see the [documentation](meshing.md) in the
```

Comment:
I'm not sure what this link is meant to lead to.  I don't see more details in `meshing.md`.